doc/Type/Mu.pod6

```diff
@@ -609,9 +609,10 @@ Returns the given item to the enclosing C<gather> block, without introducing a n
 
     method so()
 
-Returns a C<Bool> value representing the result of negating (with implicit
-conversion to C<Bool>) the expression and then again computing the negation of
-the result, thus computing the C<Bool> equivalent to the original expression.
+Evaluates the item in boolean context (and thus collapses Junctions),
+and returns the result.
```

Comment:
Maybe say *for instance* collapses functions. It's a particular instance of what could be done, no the whole extent of it. Also give some hint of what might be the result in boolean context. Double negation at least describes the process to arrive to that value. "Collapses Junctions" does not give any hint.

Reply:
Good point. That wording is copied from operator prefix:<so>, but could be better in both places. There is no such process of double negation being done, so I think it's misleading to bring up. "Collapses Junctions" is not describing how it's done, it's a significant guarantee provided by the language (i.e., after .so() you will NOT have a Junction left over waiting to be resolved elsewhere, it'll definitely be either True or False right here). I agree that it's less than clear.